src/renderer/components/opds/PageNavigation.tsx

```diff
@@ -0,0 +1,73 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import * as React from "react";
+
+import { TranslatorProps, withTranslator } from "readium-desktop/renderer/components/utils/hoc/translator";
+
+import * as ArrowRightIcon from "readium-desktop/renderer/assets/icons/baseline-arrow_forward_ios-24px.svg";
+import * as ArrowLeftIcon from "readium-desktop/renderer/assets/icons/baseline-arrow_left_ios-24px.svg";
+import SVG from "readium-desktop/renderer/components/utils/SVG";
+
+import * as styles from "readium-desktop/renderer/assets/styles/opds.css";
+
+interface Props extends TranslatorProps {
+    goto: (urld: string) => void;
+    nextUrl?: string;
+    previousUrl?: string;
+}
+
+class PageNavigation extends React.Component<Props> {
+    public constructor(props: Props) {
+        super(props);
+
+        this.handleKeyDown = this.handleKeyDown.bind(this);
+    }
+
+    public componentDidMount() {
+        document.addEventListener("keydown", this.handleKeyDown);
+    }
+
+    public componentWillUnmount() {
+        document.removeEventListener("keydown", this.handleKeyDown);
+    }
+
+    public render() {
+        const { goto, nextUrl, previousUrl, __ } = this.props;
+
+        return (
+            <div className={styles.opds_page_navigation}>
+                { previousUrl ?
+                    <button onClick={() => goto(previousUrl)}>
+                        <SVG svg={ArrowLeftIcon} />
+                        {__("opds.previous")}
+                    </button>
+                : <div/> }
+                { nextUrl ?
+                    <button onClick={() => goto(nextUrl)}>
+                        {__("opds.next")}
+                        <SVG svg={ArrowRightIcon} />
+                    </button>
+                : <div/>}
+            </div>
```

Comment:
Should this also display OPDS `metadata` `numberOfItems` and `itemsPerPage`, to give the user more pertinent pagination information?

Reply:
Example: http://readium2.herokuapp.com/opds-v1-v2-convert/https%3A%2F%2Fcatalog.feedbooks.com%2Fpublicdomain%2Fbrowse%2Fen%2Ftop.atom%3Fcat%3DFBFIC009000%26page%3D2

Reply:
I just added it in my last commit